Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -980,12 +968,14 @@ namespace OpenInfraPlatform {
 								double y = coords[1] * lengthFactor;
 								double z = coords[2] * lengthFactor;
 								loop2D[j].push_back(carve::geom::VECTOR(x, y, z));
+								return loop2D;

```

Comment:
Why is a return inside the for loop? This should be at the end of the function, probably?

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/a7a0a904dae38eedb4769899d6ab204946ee43f8